packages/svelte-query/src/__tests__/createQuery/CreateQuery.test.svelte

```diff
@@ -1,6 +1,6 @@
 <script lang="ts">
-  import { createQuery, QueryClient } from '../lib'
-  import { setQueryClientContext } from '../lib/context'
+  import { createQuery, QueryClient } from '$lib'
+  import { setQueryClientContext } from '$lib/context'
```

Comment:
I'm preparing to remove sveltekit and just use vite-plugin-svelte, can you switch this back to ../lib imports?